include/fmt/format.h

```diff
@@ -43,9 +43,13 @@
 #include <cstring>           // std::memcpy
 #include <initializer_list>  // std::initializer_list
 #include <limits>            // std::numeric_limits
-#include <stdexcept>         // std::runtime_error
-#include <string>            // std::string
-#include <system_error>      // std::system_error
+#if defined(__GLIBCXX__) && !defined(_GLIBCXX_USE_DUAL_ABI)
```

Comment:
Maybe `__GLIBCXX__ < 20150422` instead of _GLIBCXX_USE_DUAL_ABI? https://stackoverflow.com/questions/37118114/value-of-glibcxx-for-each-libstdc-release

Reply:
Comparing dates won't work:
https://github.com/fmtlib/fmt/pull/2349#discussion_r651044172